docs/distributing-an-application/installing-with-swarm.md

```diff
@@ -12,14 +12,19 @@ We distribute an installation script that can be used to install Replicated into
 
 ### Basic Install
 
-Save the Docker Compose yaml to a file and run. We recommend reading and understanding the Compose file prior to running.
```

Comment:
we are now caught up with the native scheduler for `swarm_docker_version_default`